### ISSUE TITLE
License incorrectly marked in setup.py

### ISSUE THREAD

**Issue by jamalex** [learningequality/ka-lite#4382]
Setup.py indicates that KA Lite is GPL, whereas it is actually MIT license:
https://github.com/learningequality/ka-lite/blob/0.15.x/setup.py#L355

**Comment by benjaoming** [comment]
I'm pretty sure we've discussed this already.

Maybe it has to do with the fact that we are bundling other GPL licensed libraries.

**Comment by benjaoming** [comment]
See:
https://en.wikipedia.org/wiki/License_compatibility#GPL_compatibility

**Comment by benjaoming** [comment]
Possible values are listed here: https://pypi.python.org/pypi?%3Aaction=list_classifiers

Unless you wanna go through every single license in [python-packages](https://github.com/learningequality/ka-lite/tree/master/python-packages), I'd suggest accepting this "safe side" value and instead focus our efforts on cleaning up the dependency burden.

**Comment by jamalex** [comment]
Oh, so I did actually go through python-packages, and the last time I did so, we didn't have any GPL libraries:
https://github.com/learningequality/ka-lite/blob/master/LICENSE#L38

Do you know of any that have since been added?

On the JS side, it looks like the libraries that mention GPL are "Dual licensed under the MIT or GPL":
https://github.com/learningequality/ka-lite/search?utf8=%E2%9C%93&q=gpl

If we simply depend on GPL libraries, my understanding is that doesn't force us to be GPL. However, our new "depending but then also static bundling for distribution" approach may still require GPL compliance (assuming we do actually have GPL code somewhere)?

**Comment by benjaoming** [comment]
There are ~32 packages/modules in `python-packages/`, the license file lists 15 of those. So yes, more have been added for sure. I'm afraid you're the only one who took care to fill in the LICENSE file.

Anyways, searching doesn't reveal a lot of problems. Apparently `requests` ships with `chardet` bundled, which is LGPL.. but that doesn't affect our MIT license, as long as we do not remove the license description of theirs.

```
➜  ka-lite git:(coverage) grep -r "General Public License" python-packages
python-packages/requests/packages/chardet/gb2312prober.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/chardistribution.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/sbcsgroupprober.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/langgreekmodel.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/gb2312freq.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/mbcsgroupprober.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/universaldetector.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/euctwfreq.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/constants.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/langhungarianmodel.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/charsetgroupprober.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/langcyrillicmodel.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/escsm.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/sjisprober.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/codingstatemachine.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/charsetprober.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/euckrfreq.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/big5prober.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/euctwprober.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/eucjpprober.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/mbcharsetprober.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/langhebrewmodel.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/escprober.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/utf8prober.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/jisfreq.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/euckrprober.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/langthaimodel.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/__init__.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/big5freq.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/latin1prober.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/jpcntx.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/langbulgarianmodel.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/mbcssm.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/sbcharsetprober.py:# Lesser General Public License for more details.
python-packages/requests/packages/chardet/hebrewprober.py:# Lesser General Public License for more details.
python-packages/requests/cacert.pem:# either the GNU General Public License Version 2 or later (the "GPL"), or
python-packages/requests/cacert.pem:# the GNU Lesser General Public License Version 2.1 or later (the "LGPL"),
python-packages/django/utils/baseconv.py:# This Baseconv distribution contains no GNU General Public Licensed (GPLed)
python-packages/httplib2/cacerts.txt:# either the GNU General Public License Version 2 or later (the "GPL"), or
python-packages/httplib2/cacerts.txt:# the GNU Lesser General Public License Version 2.1 or later (the "LGPL"),
```

**Comment by jamalex** [comment]
Fixed in #4384.
